server/db/model/Confirmation.js

```diff
@@ -10,24 +22,67 @@ var schema = new mongoose.Schema({
     },
     phone: {
         type: String,
-        required: true
+        required: true,
+        form: {
+            user_editable: true,
+            label: 'Phone Number',
+            placeholder: '(313) 867-5509'
+        }
     },
     graduation: {
         type: String,
-        required: true
+        enum: years,
+        required: true,
+        form: {
+            user_editable: true,
+            label: 'Graduation Year',
+            select: years.slice(0, -1).concat([end + ' or later'])
+        }
     },
     degree: {
         type: String,
-        required: true
+        enum: ['highschool', 'bachelor', 'master', 'doctorate'],
+        required: true,
+        form: {
+            user_editable: true,
+            label: 'Degree Type',
+            select: ['High School', 'Bachelors', 'Masters', 'Doctorate']
+        }
     },
     employment: {
         type: String,
-        required: true
+        enum: ['internship', 'fulltime', 'coop', 'none'],
+        required: true,
+        form: {
+            user_editable: true,
+            label: 'Job Interest',
+            select: ['Internship', 'Full Time', 'Co-op', 'None']
+        }
+    },
+    travel: {
+        type: String,
+        enum: ['bus', 'driving', 'fly', 'other'],
+        form: {
+            user_editable: true,
+            label: 'Travel',
+            select: ['MHacks Bus', 'Driving', 'Flying', 'Other']
+        }
     },
-    travel: String,
     skills: {
         type: [String],
-        default: []
+        default: [],
+        form: {
+            user_editable: true,
+            label: 'Skills',
+            array_select: skills
+        }
+    },
+    confirm_button: {
```

Comment:
I'm sure it won't actually affect us, but is there some option we can set so that this isn't stored in the db for every document